### DISCUSSION TITLE
What am I doing wrong?

### DISCUSSION THREAD

**Discussion by wollymb** [Mr-MIBonk/M.I.B._More-Incredible-Bash#46] [Q&A] [answered]
Hi all,

I recently joined the VAG community with the purchase of my 2016 Golf R. I live in Europe, while the car is from the US. When I press App-Connect, a message pops up that I need a code to activate App-connect and I need to consult my VWAG dealer about it. I looked in to this only to find out they charge €400 for the activation of App-connect.

This is how I got introduced to patching and using the MIB toolbox. Installation of the toolbox went great, everything seems to work. My firmware train is:
MHI2_US_VWG11_P2126_1
MU version: 0344

I uploaded all the patchfiles to my SD since it would automatically select the right one. When running the AIO patch, the follwoing pops up:

![thumbnail_IMG_4527](https://user-images.githubusercontent.com/116948608/198825126-e3cc2ab8-71c7-4bbe-99e7-c6d8866c2d62.jpg)

Could my unit allready have been patched? How do I return it to stock before patching? Should I have done something with the metainfo.txt before installing? Was the patch for my unit missing? All I want really, is for App-connect to work without buying a code at the dealership but apart from following the M.I.B. guide step by step my knowledgeabout this stuff is far from what is required I feel.

I have uploaded my /backup file to the M.I.B_Backup folder here so you might see what I'm doing wrong. I would really appreciate the help guys! It feels as like I'm allmost there, but just not quite.

**Comment by harman-f** [comment]
Update to latest FW 1st and patch again afterwards.

**Comment by wollymb** [comment]
Could you tell me how to? I cant find the firmware update for my train

**Reply by harman-f** [reply]
mibsolution.one

**Reply by wollymb** [reply]
There is no firmware for MHI2_US_VWG11_P2126_1 on here. No firmware for mhi2_US trains for Volkswagen at all if im not mistaken. Is there another way? Or am i just overlooking it?

I read somewhere that the firmware can be overwritten to stock using a custom metainfo.txt during installation of m.i.b. Could this be a solution?

**Reply by harman-f** [reply]
as long as you stay in MHI2_US_VWG11 you can just run the latest stock FW update. All available on mibsolution.one.

**Reply by wollymb** [reply]
The only one I can find on mibsolution.one that comes close is:
MHI2_ER_VWG11_P2125_MU0343

European, not US. Also its a lower version, 2125, whilst I have 2126_1. Can i run a European update in US unit?

**Comment by harman-f** [comment, marked as answer]
Do not mix US/EU ;)

You are right, there is no FW for US G11, was quite sure there is one.

I see the problem now!

You have MHI2_US_VWG11_**P**2126_1 and M.I.B only has patch for MHI2_US_VWG11_**S**2126_1.
For now just rename the folder S to P on SD!

Patch file is the same!

**Reply by wollymb** [reply]
Allright! I will give this a try this weekend

**Reply by wollymb** [reply]
Worked like a charm!
